evil-ex.el

```diff
@@ -229,6 +229,12 @@ Otherwise behaves like `delete-backward-char'."
   (insert result)
   (exit-minibuffer))
 
+(defun evil-ex-elisp-completion-at-point ()
+  "Complete an `evil-ex' Elisp expression."
+  (when (and (fboundp 'elisp-completion-at-point)
+             (string-prefix-p "(" (minibuffer-contents-no-properties)))
```

Comment:
`evil-ex-cmd` is probably the right thing to use here. However, if I replace `minibuffer-contents-no-properties` with it, I get an error ("Args out of range: 0, 0").